### PR TITLE
Fixing tests to use new API versions.

### PR DESCRIPTION
**Pull request by chiragg4u**
Fixing tests to include the new API version; this version is exactly same as preview version and its being stable.

### CODE REVIEW

**Comment by chiragg4u** [comment]
@demyanenko can you please review from PG team.

File: src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <PackageId>Microsoft.Azure.Management.ResourceGraph</PackageId>
     <Description>Provides Resource Graph operations for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.ResourceGraph</AssemblyName>
-    <Version>0.10.0-preview</Version>
+    <Version>1.0.0</Version>
```
Comment (dsgouda) [inline src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: @chiragg4u what are the changes included in this version? Was the code regenerated for the new API version?

Comment (chiragg4u) [reply src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: There is no change in the contract; only new API version is being published as we are getting into GA of our service.
And yes the code is regenerated by the swagger update as captured @  https://github.com/Azure/azure-rest-api-specs/pull/5432

File: src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
     <PackageTags>Azure;ResourceGraph;</PackageTags>
     <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: Please update `PackageReleaseNotes`
Please update version number in AssemblyInfo.cs

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please use Start-AutorestCodeGenerationWithLocalConfig cmdlet to generate the code using spec that is not merged yet.

File: src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj (hunk 1 above)
Comment (dsgouda) [reply src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: @chiragg4u change in API version will definitely produce a diff in the generated clients, please regenerate the code,

Comment (chiragg4u) [reply src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: Regenerated code already merged; I merged with master so all the changes are present in this PR now.

**Comment by dsgouda** [comment]
@chiragg4u I assume you are referring to https://github.com/Azure/azure-sdk-for-net/pull/5519

**Comment by dsgouda** [comment]
Please join the azure github org [here](https://aka.ms/azuregithub)

File: src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj (hunk 2 above)
Comment (chiragg4u) [reply src/SDKs/ResourceGraph/Management/Management.ResourceGraph/Microsoft.Azure.Management.ResourceGraph.csproj]: Updated will push the new changes.

**Comment by chiragg4u** [comment]
> Please use Start-AutorestCodeGenerationWithLocalConfig cmdlet to generate the code using spec that is not merged yet.

I've merged the changes from the main branch and fixed the tests; please review the changes and enable the tests back.

**Comment by chiragg4u** [comment]
Mistakenly closed the branch.

**Review by dsgouda — APPROVED** [review]
LGTM
Confirmed tests are fixed [here](https://azuresdkci.westus2.cloudapp.azure.com/view/NetCore-SDK/job/NetCore-SDK-Sign/494/console)

**Pull request merged**